src/NuGet.Core/NuGet.Test.Utility/SimpleTestSetup/ProjectFileUtils.cs

```diff
@@ -68,8 +69,15 @@ public static void AddProperty(XDocument doc, string propertyName, string proper
 
             foreach (var pair in properties)
             {
-                var subItem = new XElement(XName.Get(pair.Key, ns.NamespaceName), pair.Value);
-                entry.Add(subItem);
+                if (pair.Key.Equals("version", StringComparison.OrdinalIgnoreCase))
```

Comment:
no need for this once you start adding version as attribute instead of properties